apps/spruce/cypress/integration/image/build_information.ts

```diff
@@ -6,4 +6,41 @@ describe("build information", () => {
       cy.contains("ubuntu1804-workstation");
     });
   });
+
+  describe("packages", () => {
+    it("should show the corresponding packages", () => {
+      cy.visit("/image/ubuntu2204");
+      cy.dataCy("packages-table-row").should("have.length", 10);
+    });
+
+    it("should show different package on new page", () => {
+      cy.visit("/image/ubuntu2204");
+      cy.dataCy("packages-table-row").should("have.length", 10);
+      cy.dataCy("packages-table-row")
+        .first()
+        .find("td")
+        .first()
+        .invoke("text")
+        .then((firstPackageName) => {
+          cy.get("[data-testid=lg-pagination-next-button]").click();
+          cy.dataCy("packages-table-row").should("have.length", 10);
+          cy.dataCy("packages-table-row")
+            .first()
+            .find("td")
+            .first()
+            .invoke("text")
+            .should((nextPackageName) => {
+              expect(nextPackageName).not.to.equal(firstPackageName);
+            });
+        });
```

Comment:
I think it might be clearer to write it like this, including comments
```suggestion
      // First package name on first page.
      cy.dataCy("packages-table-row")
        .first()
        .find("td")
        .first()
        .invoke("text")
        .as("firstPackageName", { type: "static" });

      cy.get("[data-testid=lg-pagination-next-button]").click();

      // First package name on second page.
      cy.dataCy("packages-table-row")
        .first()
        .find("td")
        .first()
        .invoke("text")
        .as("nextPackageName", { type: "static" });

      // Package names should not be equal.
      cy.get("@firstPackageName").then((firstPackageName) => {
        cy.get("@nextPackageName").then((nextPackageName) => {
          expect(nextPackageName).not.to.equal(firstPackageName);
        });
      });
```